src/plugins/interactive_setup/server/plugin.ts

```diff
@@ -92,8 +97,10 @@ export class InteractiveSetupPlugin implements PrebootPlugin {
           this.#logger.debug(
             'Starting interactive setup mode since Kibana cannot to connect to Elasticsearch at http://localhost:9200.'
           );
-          const serverInfo = core.http.getServerInfo();
-          const url = `${serverInfo.protocol}://${serverInfo.hostname}:${serverInfo.port}`;
+          const { code } = this.#verificationCode;
+          const pathname = core.http.basePath.prepend('/');
+          const { protocol, hostname, port } = core.http.getServerInfo();
+          const url = `${protocol}://${hostname}:${port}${pathname}?code=${code}`;
```

Comment:
The flow is that we print the code to the terminal when one is required but hasn't been sent with the configure/enroll request:

1. User starts Kibana
2. User submits enrollment token (Server returns 403 since no code was provided but is required, when this happens, the server prints the verification code)
3. Kibana UI asks for verification code in modal (See screenshot)
4. User copies verification code from Kibana server
5. Kibana shows enrollment progress and redirects to Login page

This ensures that we only ask for the verification code when actually required.

Reply:
Okay, that's fine. Currently we don't explain where to get this code in UI, but we'll do that somehow before GA, so maybe it's just me.

Reply:
We say "Copy the verification code from Kibana server" but I agree that it would be useful to add more helpers.